nle/scripts/play.py

```diff
@@ -100,10 +112,15 @@ def play(env, mode, ngames, max_steps, seeds, savedir, no_render, render_mode, d
     while True:
         if not no_render:
             if not is_raw_env:
-                print("Previous reward:", reward)
-                if action is not None:
-                    print("Previous action: %s" % repr(env._actions[action]))
+                print("--------")
+                print(f"Previous reward: {str(reward):64s}")
+                act_str = repr(env._actions[action]) if action is not None else ""
+                print(f"Previous action: {str(act_str):64s}")
+                print("--------")
                 env.render(render_mode)
+                print("--------")
+                if not print_all_frames:
```

Comment:
That's a bit of a weird name for what it does (going back to override previous frames)?

Reply:
if you are _not_ printing all frames, then you are going back and overwriting instead.  i wanted a `store_true` parameter that would stop the overwriting behaviour. howabout `print_frames_separately`?